cartofacade/cgo_api.go

```diff
@@ -0,0 +1,400 @@
+// Package cartofacade provides an api to call into c code
+package cartofacade
```

Comment:
```suggestion
//nolint:lll
// Package cartofacade provides an api to call into c code
package cartofacade
```
Add this to disable line length linting